openedx/openedx-platform
# [BB-1840] Ironwood SSO fixes

This PR cherry-picks an existing ginkgo customization code drift required for SSO to work and also adds `gender` to the list of fields synced by the platform from SSO user data.

**Testing instructions**:
* Deploy this branch.
* Also deploy the changes to the `edx-oauth-client` from the corresponding PR branch.
* Create an account on the Drupal frontend, log in and navigate to the LMS.
* Verify that the user is logged into the LMS and all the user data is copied properly to the corresponding fields in the `User` model and the `UserProfile` model.
